v1-2/learn/how-to-write-secure-ballerina-code.md

```diff
@@ -647,20 +653,21 @@ The `oauth2:IntrospectionServerConfig` record should be provided into the `oauth
 ```ballerina
 import ballerina/http;
 import ballerina/oauth2;
+import ballerina/config;
 
 oauth2:InboundOAuth2Provider oauth2Provider = new({
     url: "https://localhost:9196/oauth2/token/introspect",
     tokenTypeHint: "access_token"
 });
-http:BearerAuthHandler oauth2Handler = new(oauth2Provider);
+http:BearerAuthHandler oAuth2Handler = new(oauth2Provider);
```

Comment:
Shall we keep that as `oauth2Handler` since that is the convention we followed.

Reply:
Ack